examples/generate_image_dataset/user_config.yaml.example

```diff
@@ -19,6 +19,19 @@ SIMULATION_CONTROLLER:
   TASK: "NullTask"
   CUSTOM_UNREAL_CONSOLE_COMMANDS: []
 
+  NAVMESH:
+    TILE_POOL_SIZE: 1024
+    TILE_SIZE_UU: 1000.0
+    CELL_SIZE: 1.0
+    CELL_HEIGHT: 1.0
+    AGENT_RADIUS: 50.0
+    AGENT_HEIGHT: 200.0
+    AGENT_MAX_SLOPE: 1.0
+    AGENT_MAX_STEP_HEIGHT: 1.0
+    MIN_REGION_AREA: 0.0
+    MERGE_REGION_SIZE: 400.0
+    MAX_SIMPLIFICATION_ERROR: 1.3
+
   CAMERA_AGENT:
     CAMERA_ACTOR_NAME: "CameraActor"
```

Comment:
`"camera_actor"` to match our convention in `getting_started`.

Reply:
Updated